packages/phoenix-event-display/src/tests/managers/three-manager/scene-manager.test.ts

```diff
@@ -0,0 +1,235 @@
+/**
+ * @jest-environment jsdom
+ */
+import { SceneManager } from '../../../managers/three-manager/scene-manager';
+import { Object3D, Scene, Vector3 } from 'three';
+import {
+  AmbientLight,
+  AxesHelper,
+  BoxGeometry,
+  BufferGeometry,
+  Camera,
+  Color,
+  DirectionalLight,
+  Group,
+  Mesh,
+  MeshBasicMaterial,
+} from 'three';
+import { CoordinateHelper } from '../../../helpers/coordinate-helper';
+
+describe('SceneManager', () => {
+  let sceneManager: SceneManager;
+  const ignoreList = [
+    new AmbientLight().type,
+    new DirectionalLight().type,
+    new AxesHelper().type,
+  ];
+
+  beforeEach(() => {
+    sceneManager = new SceneManager(ignoreList, false);
+  });
+
+  it('should update position of directional light for each frame rendered', () => {
+    sceneManager.updateLights(new Camera());
+    expect(sceneManager.cameraLight).toBeUndefined();
+  });
+
+  it('should get a clean copy of the scene', () => {
+    const scene = new Group();
+    scene.add(new Mesh(new BoxGeometry(), new MeshBasicMaterial()));
+    const cleanScene = sceneManager.getCleanScene();
+
+    expect(cleanScene).toBeInstanceOf(Scene);
+    expect(cleanScene.children.length).toBe(0);
```

Comment:
We need to check here that the clean scene doesn't contain the ignored objects type. No need to check for `up`, `position` and `scale` properties.